packages/flutter/lib/src/rendering/selection.dart

```diff
@@ -142,6 +142,10 @@ mixin Selectable implements SelectionHandler {
   /// The size of this [Selectable].
   Size get size;
```

Comment:
Is this still neededed if we we use boundingBoxes to locate the selectable?

Reply:
The only reason I can think of for keeping this is that it is a nice convenience for the user to get the size of the Selectable without merging their own boxes. In the framework I think there's only one more usage of it, but that can be replaced by merging the boxes and getting the size. I'm leaning more towards keeping it, but don't feel strongly either way. What do you think?

Reply:
I don't have strong preference as well. I am ok if we keep it for now. There is another project to implement search feature that may require more refactoring on the renderparagraph, we can decide whether we want to keep this field when we do that.